examples/custom-beacon-withdrawals/src/main.rs

```diff
@@ -12,26 +12,33 @@ use alloy_evm::{
 use alloy_sol_macro::sol;
 use alloy_sol_types::SolCall;
 use reth::{
-    api::{ConfigureEvm, NodeTypes},
-    builder::{components::ExecutorBuilder, BuilderContext, FullNodeTypes},
+    builder::{components::ExecutorBuilder, BuilderContext},
     cli::Cli,
-    providers::BlockExecutionResult,
+    primitives::SealedBlock,
     revm::{
         context::{result::ExecutionResult, TxEnv},
         db::State,
         primitives::{address, hardfork::SpecId, Address},
         DatabaseCommit,
     },
```

Comment:
I think we could re-export this as

```rust
pub use reth_evm::revm as revm;

```

here:

https://github.com/paradigmxyz/reth/blob/19f8070565bd4bedc4b06d113b4fbc249410c875/crates/ethereum/reth/src/lib.rs#L41-L49

for both reth-ethereum and reth-op